.github/workflows/ci.yml

```diff
@@ -12,6 +12,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
```

Comment:
It appears to be fairly common to use a GH action "matrix" to run each job against multiple go versions; but I think we're find just using the version in the Dockerfile for now?